static/docs/tutorials/deep/preparation.md

```diff
@@ -49,17 +49,6 @@ $ echo ".env/" >> .gitignore
 $ pip install -r code/requirements.txt
 ```
 
-## Install DVC
-
-You'll need DVC on your system to continue this tutorial. Please refer to
-[Installation](/doc/install) for the easiest way to install DVC on your system.
-For this tutorial, installing DVC with [pip](https://pip.pypa.io/) is probably
-the easiest:
```

Comment:
Thanks a lot @welenofsky, this would've been correct per the issue's original suggestion. However as noted in https://github.com/iterative/dvc.org/issues/760#issuecomment-548185227 we decided not to do this. I'm also updating this in https://github.com/iterative/dvc.org/pull/763/files BTW... Will think about how to use your PR though. Let me think...

Reply:
See https://github.com/iterative/dvc.org/pull/761#issuecomment-548561984